### DISCUSSION TITLE
Execute current query in sqllab

### DISCUSSION THREAD

**Discussion by cploonker** [apache/superset#18514] [Ideas]
### Is your feature request related to a problem? Please describe.

In superset sqllab i am able to execute just one query (out of multiple in the window) by selecting that entire query and using the option "run selection".  I want to be able to execute the current query(delimited by semicolon on both sides) without re-selecting after every small change.

### Describe the solution you'd like

User should separate each query by a delimiter e.g. semi-colon.
Superset can then identify the current query using cursor position. Query is the text between the delimiters just before and just after the cursor.
Enable a shortcut Ctrl + Option + Enter to execute the current query where the cursor is.

### Describe alternatives you've considered

I have been selecting the query and using "run selection" option which is more clicks.

**Comment by rumbin** [comment]
I second this. It is the default behaviour of many other SQL editors, like, e.g. DBeaver

**Comment by rumbin** [comment]
Such editors have the default behaviour inverted w.r.t. Superset:

Crtl+Enter runs the code at the cursor position, i.e. between two semi-colons.

If more statements are to be run in a row, the user needs to select all text that should be executed.
